### PR TITLE
 Add orange-widget-base recipe to conda-forge

### PR DESCRIPTION
**Pull request by ales-erjavec**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/orange-widget-base/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  doc_url: https://orange-widget-base.readthedocs.io/
+  dev_url: https://github.com/biolab/orange-widget-base/
+
+extra:
+  recipe-maintainers:
+    - ales-erjavec
+    - lanzagar
+    - markotoplak
+    - primozgodec
```
Comment (ales-erjavec) [inline recipes/orange-widget-base/meta.yaml]: @lanzagar @markotoplak @primozgodec can you confirm you would like to be maintainers of this recipe

Comment (lanzagar) [reply recipes/orange-widget-base/meta.yaml]: I confirm.

Comment (PrimozGodec) [reply recipes/orange-widget-base/meta.yaml]: I confirm

Comment (markotoplak) [reply recipes/orange-widget-base/meta.yaml]: I confirm.

**Comment by PrimozGodec** [comment]
ping @conda-forge/staged-recipes

File: recipes/orange-widget-base/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+{% set version = "4.0.0" %}
+{% set sha256 = "0f080a410f6aa9c929d8f7f22b7157ac3d53ca0d34de10026dedaaedad46b8e9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (chrisburr) [inline recipes/orange-widget-base/meta.yaml]: As it's only used in one place, putting the hash here makes things easier for the update bot:
```suggestion
  sha256: 0f080a410f6aa9c929d8f7f22b7157ac3d53ca0d34de10026dedaaedad46b8e9
```

File: recipes/orange-widget-base/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{% set name = "orange-widget-base" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "0f080a410f6aa9c929d8f7f22b7157ac3d53ca0d34de10026dedaaedad46b8e9" %}
```
Comment (chrisburr) [inline recipes/orange-widget-base/meta.yaml]:
```suggestion
```

**Pull request merged**